content/docs/command-reference/build.md

```diff
@@ -6,11 +6,16 @@ Python package.
 ## Synopsis
 
 ```usage
-usage: mlem build [options] model [subtype]
+Usage: mlem build builder [options] model
 
-arguments:
-MODEL      Path to model  [required]
-[SUBTYPE]  Type of build. Choices: ['whl', 'pip', 'docker_dir', 'docker']
+Builtin builders:
```

Comment:
MODEL -> model

required / optional check how it's done in other docs

list with options - it was better the way it was done (compact, easier to read, etc)

Reply:
required and optional are different things: required means you always should provide it, optional means it can have `None` value. Required optional means you will get error if you don't provide it, but you can set it to `None`. Not required not optional means there is a default value which is not `None` and you can't set it to `None`

also cli changed a bit, now `mlem build *` are subcommands (instead of argument). Also `mlem build` is a separate command (for building from saved config using `-l`). I will update this soon when I decide how to better reflect this (same with `serve` and `apply-remote`)